ground/gcs/src/plugins/uploader/bl_messages.h

```diff
@@ -85,6 +85,103 @@ enum bl_commands {
  * Note: These enum values MUST NOT be changed or backward
  *       compatibility will be broken
  */
+#ifdef _MSC_VER
```

Comment:
@guilhermito If it would help, I could take a pass at de-duplicating this and send you a pull request to your branch.